ipatests/test_integration/test_acme.py

```diff
@@ -542,3 +547,130 @@ def install(cls, mh):
 
         tasks.install_client(cls.master, cls.clients[0])
         tasks.install_replica(cls.master, cls.replicas[0])
+
+
+class TestACMERenew(IntegrationTest):
+
+    num_clients = 1
+
+    @classmethod
+    def prepare_acme_client(cls):
+        # cache the acme service uri
+        acme_host = f'{IPA_CA_RECORD}.{cls.master.domain.name}'
+        cls.acme_server = f'https://{acme_host}/acme/directory'
+
+        # install acme client packages
+        if not skip_certbot_tests:
+            tasks.install_packages(cls.clients[0], ['certbot'])
+        if not skip_mod_md_tests:
+            tasks.install_packages(cls.clients[0], ['mod_md'])
+
+    @classmethod
+    def install(cls, mh):
+
+        # install packages before client install in case of IPA DNS problems
+        cls.prepare_acme_client()
+
+        tasks.install_master(cls.master, setup_dns=True)
+        tasks.install_client(cls.master, cls.clients[0])
+
+    @pytest.fixture
+    def issue_and_expire_cert(self):
+        """Fixture to expire cert by moving date past expiry of acme cert"""
+        # enable the ACME service on master
+        self.master.run_command(['ipa-acme-manage', 'enable'])
+
+        # register the account with certbot
+        # clean up any existing registration and certificates
+        self.clients[0].run_command(
+            [
+                'rm', '-rf',
+                '/etc/letsencrypt/accounts',
+                '/etc/letsencrypt/archive',
+                '/etc/letsencrypt/csr',
+                '/etc/letsencrypt/keys',
+                '/etc/letsencrypt/live',
+                '/etc/letsencrypt/renewal',
+                '/etc/letsencrypt/renewal-hooks'
+            ]
+        )
+        # service is enabled; registration should succeed
+        self.clients[0].run_command(
+            [
+                'certbot',
+                '--server', self.acme_server,
+                'register',
+                '-m', 'nobody@example.test',
+                '--agree-tos',
+                '--no-eff-email',
+            ],
+        )
+
+        # request a standalone acme cert
+        self.clients[0].run_command(['systemctl', 'stop', 'httpd'])
+        self.clients[0].run_command(
+            [
+                'certbot',
+                '--server', self.acme_server,
+                'certonly',
+                '--domain', self.clients[0].hostname,
+                '--standalone',
+            ],
+        )
```

Comment:
It's a copy-paste from test_certbot_certonly_standalone. Create a method outside of the class and call it from here.